<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>756</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>756</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>756</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Netflix</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../USER LOGIN/loogo.jpg</normaloff>../USER LOGIN/loogo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border-radius:5px;
}

QPushButton:hover{
color:rgb(255,255,255);
}

QPushButton:pressed{
background-color:rgb(255,255,255);
}
</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Don't have an account yet?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>360</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border-radius:5px;
}

QPushButton:hover{
color:rgb(255,255,255);
}

QPushButton:pressed{
background-color:rgb(255,255,255);
}
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>netflix logo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Invalid username or password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
